src/types.js

```diff
@@ -13,8 +13,10 @@ function applyRecord(columnMetadata, record) {
             /^(\d{1,4})-(\d{1,2})-(\d{1,2}) (\d{1,2}):(\d{1,2}):(\d{1,2})(?:.(\d{1,3}))?(?:\d{1,3})?$/,
           );
 
+          // eslint-disable-next-line no-case-declarations
+          const millisecondPadded = millisecond === undefined ? '000' : millisecond.padEnd(3, 0);
```

Comment:
Maybe we could default the millisecond to '' on line 12 instead? Maybe I'll just merge this one and make that small change in a separate commit